.github/workflows/triage-labeled-issues.yml

```diff
@@ -0,0 +1,31 @@
+name: Triages labeled issues by adding them to a specific project field
+
+on:
+  issues:
+    types: [ labeled ]
+
+jobs:
+  move-ts-issues:
+    if: ${{contains(github.event.label.name, format('area{0} typescript', ':'))}}
+    uses: ./.github/workflows/move-issue-to-project-and-set-fields.yml
+    with:
+      field: 'Area'
+      field_option: '🟦 Typescript'
+      project_number: '65'
+    secrets: inherit
+  move-migration-issues:
+    if: ${{contains(github.event.label.name, format('area{0} migration ➡️', ':'))}}
+    uses: ./.github/workflows/move-issue-to-project-and-set-fields.yml
+    with:
+      field: 'Area'
+      field_option: '🗺 Migration'
+      project_number: '65'
```

Comment:
Not sure if it’s worth storing this as an environment variable[^1]. I think adding more labels to our workflows will be at least somewhat manual so maybe lower value.

[^1]: https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow

Reply:
I did try, but got an error about using environment variables for [workflow_call inputs](https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputs).
![image](https://github.com/carbon-design-system/ibm-products/assets/10215203/89e2db3c-251d-4035-be2c-d815fb2e55c7)